examples/airline/configs/tools.py

```diff
@@ -0,0 +1,28 @@
+def escalate_to_agent(reason=None):
+    return f"Escalating to agent: {reason}" if reason else "Escalating to agent"
```

Comment:
Refactor the `escalate_to_agent` function to handle different escalation reasons more robustly, possibly by implementing different escalation strategies based on the reason. [medium]